pkg/apiserver/event/sync/store.go

```diff
@@ -42,9 +43,10 @@ func StoreProject(ctx context.Context, name string, ds datastore.DataStore, proj
 		return err
 	}
 	if projectService != nil {
+		caser := cases.Title(language.English)
```

Comment:
Interesting fix. Related to this issue, https://github.com/golang/go/issues/48367.